packages/ckeditor5-upload/docs/_snippets/features/base64-upload.js

```diff
@@ -18,6 +18,14 @@ ClassicEditor
 	} )
 	.then( editor => {
 		window.editor = editor;
+
+		// eslint-disable-next-line no-undef
```

Comment:
Instead of disabling a rule add setTimeout to known globals in this file.

Reply:
Looks like it was added.

Reply:
With my latest commit I've also corrected this and other comments, sorry for not communicating this :)